src/common/Rating/styled.js

```diff
@@ -0,0 +1,76 @@
+import styled, {css} from "styled-components";
+import { ReactComponent as star} from "../../common/img/star.svg";
+
+export const Wrapper = styled.div`
+  display: flex;
+  gap: 12px;
+  flex-wrap: wrap;
+  align-items: center;
+  line-height: 1.5;
+  font-size: 16px;
+  height: 100%;
+  align-content: flex-end;
+  `;
+
+export const Vote = styled.span`
+color: ${({ theme }) => theme.colors.woodsmoke};
+font-weight: bold;
+`;
+
+export const Votes = styled.span`
+  color: ${({ theme }) => theme.color.waterloo};
```

Comment:
@Dawid-Zieba you missed "S" in this code, correct it to: `color: ${({theme }) => theme.colors.waterloo};` :)